os/drivers/serial/serial.c

```diff
@@ -109,6 +110,15 @@ static int uart_poll(FAR struct file *filep, FAR struct pollfd *fds, bool setup)
  * Private Variables
  ************************************************************************************/
 
+#define CONFIG_LOG_CHUNK_SIZE 1024
```

Comment:
I think it is better to make config for these defines.
Then we can easily change them through menuconfig.